src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -653,6 +670,21 @@ private ImmutableArray<ParameterSymbol> ComputeParameters(DiagnosticBag diagnost
             return parameters.ToImmutableAndFree();
         }
 
+        internal override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes)
+        {
+            base.AddSynthesizedReturnTypeAttributes(moduleBuilder, ref attributes);
+
+            var compilation = this.DeclaringCompilation;
+            if ((ReturnTypeFlowAnalysisAnnotations & FlowAnalysisAnnotations.MaybeNull) != 0)
+            {
+                AddSynthesizedAttribute(ref attributes, new SynthesizedAttributeData(_property.MaybeNullAttributeIfExists));
```

Comment:
>new SynthesizedAttributeData(_property.MaybeNullAttributeIfExists) [](start = 56, length = 66)

Both attributes should be located and added in one pass through property's attributes